tracing-opentelemetry/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "tracing-opentelemetry"
-version = "0.4.0"
+version = "0.5.0"
```

Comment:
Let's make sure there are not also references to the prior version in the README and in RustDoc before merging this.

Reply:
Checked, shouldn't have any 0.4.0 in the docs 👍